samples/snippets/core/deploying/deployment-example.cs

```diff
@@ -0,0 +1,36 @@
+using System;
```

Comment:
As I stated up at the start, this isn't necessary to teach these concepts or show how the tooling works to create these deployment types. Therefore, I suggest removing this and just going with a good 'ole default "Hello World!" console app.